### ISSUE TITLE
Proposed links inside tables 

### ISSUE THREAD

**Issue by carlrobert** [vschroeter/obsidian-virtual-linker#36]
Insert a table and add known note names as headings or rows. Both headings and rows are recognized by virtual linker.
None of the entries can be converted to real links. Nothing happens when you try do this operation via the context (popup) menu.

In my vault, both vector and function are pre-existing notes.

```
| vector | function |
| ------ | -------- |
| vector | function |
```

**Comment by vschroeter** [comment]
The new tables in obsidian render their contents like in reading mode. 
This brings several problems:
- I cannot determine the position of the link in its source (since the preview rendering happens for each cell seperately, independet of the note's source document)
- I cannot determine if a virtual link inside a table was previously created in the document to avoid mutiple linking

At the moment I stilll have no good idea on how to fix it

**Comment by vschroeter** [comment]
I have deactivated in Fixed in [Version 1.4.7](https://github.com/vschroeter/obsidian-virtual-linker/releases/tag/1.4.7) the conversion action in this case for the time being until we found a solution.

**Comment by carlrobert** [comment]
OK, no promises but will keep my eyes open for a solution

**Comment by TempaYeshe** [comment]
Is this the reason "avoid linking in current line" does not work in tables?

**Comment by vschroeter** [comment]
Yes, I guess so. This may be all related to the problem, that tables are rendered with the reading mode renderer.
